plugins/config/lib/configService.js

```diff
@@ -536,7 +536,7 @@ function createMissingFoldersAsync(lastPath, pluginID, currentResource, resource
   };
 
   var createUserDirectory = function() {
-    logger.debug("Set lastPath to "+lastPath);
+    logger.debug("ZWED0224I", lastPath); //logger.debug("Set lastPath to "+lastPath);
```

Comment:
Was  "ZWED0224I":"Set lastPath to %",
should be  "ZWED0224I":"Set lastPath to %s",

Reply:
addressed.